ArgoTests/Tests/PListDecodingTests.swift

```diff
@@ -20,5 +20,6 @@ class PListDecodingTests: XCTestCase {
     XCTAssert(model?.eStringArrayOpt?.count == 0)
     XCTAssert(model?.userOpt != nil)
     XCTAssert(model?.userOpt?.id == 6)
+    XCTAssert(model!.dict == ["foo": "bar"])
```

Comment:
these probably need to be `!` instead of `?` because native Swift comes with `==` implementations of `Optional<Type> == Type` for all it's native types. They probably didn't have one for `Dictionary` ... might be able to just wrap the Dictionary in `.Some` or `pure` if Runes is imported above.